lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+
+class Turn
+    attr_reader :cards, :turns
```

Comment:
indent this back a bit, so it's in line with where `def` is on your method declarations below.